SignalServiceKit/src/Protos/Generated/SSKProto.swift

```diff
@@ -220,10 +238,29 @@ public enum SSKProtoError: Error {
     fileprivate let proto: SignalServiceProtos_Content
 
     @objc public let dataMessage: SSKProtoDataMessage?
+    @objc public var hasDataMessage: Bool {
```

Comment:
Why do we have all the hasFoo methods for reference types?

Reply:
Because we use them all over the place, and these PRs are already huge.  I've made a card to migrate away from them.